openreview/conference/builder.py

```diff
@@ -1137,14 +1137,14 @@ def set_impersonators(self, emails = []):
 
         return self.webfield_builder.set_impersonate_page(self, impersonate_group)
 
-    def setup_matching(self, committee_id=None, affinity_score_file=None, tpms_score_file=None, elmo_score_file=None, build_conflicts=None, alternate_matching_group=None):
+    def setup_matching(self, committee_id=None, affinity_score_file=None, tpms_score_file=None, elmo_score_file=None, build_conflicts=None, alternate_matching_group=None, scores_stream=None, compute_scores=False):
```

Comment:
I would like to merge affinity_score_file, score_stream and compute_scores in a single parameter, could we use the type of the value to know that?

let's call it `compute_affinity_scores`:

- if `compute_affinity_scores` is String then we assume it is a file name and we should read the scores from the file.
- if `compute_affinity_scores` is a Stream then we read the scores from the stream.
- if `compute_affinity_scores` is  boolean and is True then we read the scores from the API.
- if `compute_affinity_scores` is boolean and is False then don't compute/read any scores. 


Does it work?

I know that removing the `affinity_score_file` parameter from the function can break a lot of things. I would suggest to mark this function as deprecated and create a new one:

`def setup_committee_matching(self, committee_id, compute_scores=False, compute_conflicts=False, alternate_matching_group)`

I'm open to other function names.